src/app/index.tsx

```diff
@@ -1,3 +1,4 @@
 export { Header } from './header/Header';
 export { Content } from './content/Content';
 export { SidebarContainer } from './sidebar/containers/SidebarContainer';
+export { Feedback } from './misc/Feedback';
```

Comment:
I think it should be in a `feedback` directory imo (misc can get carried away and act as a "catch-all")